screens/YourCart.js

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image, FlatList} from 'react-native';
+import PrimaryButton from './../components/PrimaryButton';
+
+export default class YourCart extends React.Component {
```

Comment:
Please stop just using the Figma CSS they are not dynamic. In your implementation, there's a lot of whitespace which is ultimately wasted space. Make the blue and purple box expand with the height of the screen.

Also, the pencil should be closer to the right edge of the boxes - look back at the mock as reference